src/pages/_app.tsx

```diff
@@ -237,8 +249,17 @@ function MyApp({ Component, pageProps }: AppPropsWithLayout) {
 
 		_logger.info({ scwAddress }, 'got scw address')
 
-		setBiconomyWalletClients(prev => ({ ...prev, [chainId]: _biconomyWalletClient }))
-		setBiconomyDaoObjs((prev: any) => ({ ...prev, [chainId]: _biconomy }))
+		let localBiconomyClients: typeof biconomyWalletClients = {}
+		setBiconomyWalletClients(prev => {
```

Comment:
this function may or may not be called immediately -- can create weird bugs if you set it like this. Should simply return the newly created biconomy obj -- nothing more